inference/mlflow-huggingface/src/models/bert-base-uncased/spec.yaml

```
@@ -0,0 +1,3 @@
+name: bert-base-uncased
+path: C:\Users\Administrator\azureml-assets\scripts\azureml-assets\tmp\bert-base-uncased
```

The full path should be removed to minimize confusion. How about this?
```suggestion
path: will-be-replaced-when-model-is-published
```

---

That's fine!